Tribler/Main/tribler_main.py

```diff
@@ -419,6 +419,65 @@ def PostInit2(self):
 
         startWorker(None, self.loadSessionCheckpoint, delay=5.0, workerType="ThreadPool")
 
+    def startAPI(self, session):
```

Comment:
This method got removed completely in my (now merged) PR so maybe something went wrong during the rebase.